packages/gatsby/src/utils/worker/child/schema.ts

```diff
@@ -1,8 +1,19 @@
+import apiRunnerNode from "../../api-runner-node"
+import { build } from "../../../schema"
 import { setState } from "./state"
 
 export function setInferenceMetadata(): void {
   setState([`inferenceMetadata`])
 }
+
 export function setQueries(): void {
   setState([`components`, `staticQueryComponents`])
 }
+
+export async function buildSchema(): Promise<void> {
+  setInferenceMetadata()
+
+  await apiRunnerNode(`createSchemaCustomization`)
+
+  await build({ fullMetadataBuild: false, parentSpan: {} })
+}
```

Comment:
maybe consider adding some checking if config was loaded? as all API running wouldn't really work without plugins loaded

might not be needed for this PR, but we definitely would want some checks to make sure we have everything that is needed to actually build schema ready